components/i2c/include/i2c_defs.h

```diff
@@ -0,0 +1,16 @@
+#ifndef _I2C_DEFS_H_
+#define _I2C_DEFS_H_
+
+#include "esp_err.h"
+#include "driver/i2c.h"
+
+#define I2C_MASTER_SCL_IO CONFIG_I2C_MASTER_SCL_IO // GPIO number for I2C master clock
+#define I2C_MASTER_SDA_IO CONFIG_I2C_MASTER_SDA_IO // GPIO number for I2C master data
+#define I2C_MASTER_NUM I2C_NUM_0                   // I2C port number for master dev
+#define I2C_MASTER_FREQ_HZ 100000                  // I2C master clock frequency (100kHz)
```

Comment:
The better place to define the frequency is a Kconfig - then you can easily change this value in Menuconfig.

Reply:
Done, moved it